.github/workflows/tests.yml

```diff
@@ -34,7 +34,7 @@ jobs:
         pip install -e ".[shell]"
         pip install -r requirements/requirements-test.txt
         pip install -r requirements/requirements-rotbaum.txt
-        pip install -r requirements/requirements-extras-m-competitions.txt
```

Comment:
Why remove the `m-competition` requirement?

Reply:
It was a mistake when I resolve the conflicts in branch updating. I will correct it in the next commit.